index.bs

```diff
@@ -564,7 +562,8 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 {{CredentialsContainer/create()|navigator.credentials.create()}} to request the creation of a new [=credential key pair=]
 and {{PublicKeyCredential}}, managed by an [=authenticator=].
 On success, the returned {{promise}} will be resolved with a {{PublicKeyCredential}} containing an
-{{AuthenticatorAttestationResponse}} object.
+{{AuthenticatorAttestationResponse}} object. This method can be <dfn id="concept-webauthn-terminated">terminated</dfn> 
+with flag [=AbortSignal/aborted flag=], which is unset unless otherwise specificed.
```

Comment:
if we are modeling our termination and aborting functionality on [fetch](https://fetch.spec.whatwg.org), then this -- using [=AbortSignal/aborted flag=] here -- does not appear to be correct. In [the fetching alg](https://fetch.spec.whatwg.org/#fetching), `aborted` is a simple flag local to the fetch algorithm:
`... <dfn export for=fetch id=concept-fetch-terminate>terminated</dfn> with flag <var>aborted</var> ...`